conftest.py

```diff
@@ -37,6 +37,12 @@ def pytest_addoption(parser):
         default=CONFIG['API_URL'],
         help="The URL to use for running tests against."
     )
+    parser.addoption(
+        '--api-type',
+        dest="api_type",
+        default=CONFIG['API_TYPE'],
+        help="The API to test against (available: default, nominatim)."
```

Comment:
Would it make sense to use `geocodejson` instead of `default` ?

Reply:
I admit that `default` is a bit lame but `geocodejson` isn't really correct either. geocodejson is the output format. Nominatim produces that as well and that is what we still check against. What the API proxy does is to adapt the input parameters as necessary. In the particular case of Nominatim, I need to add `format=geocodejson&addressdetials=1` to the query to get the output that the tester understands. But the proxy allows us also to convert entire parameters.

Reply:
Makes sense, thanks for the clarification!